src/wallet.js

```diff
@@ -113,9 +113,11 @@ const wallet = {
     // Save in storage the encrypted private key and the hash of the pin and password
     let access = {
       mainKey: encryptedData.encrypted.toString(),
-      hash: encryptedData.hash.toString(),
+      hash: encryptedData.hash.key.toString(),
+      salt: encryptedData.hash.salt,
       words: encryptedDataWords.encrypted.toString(),
-      hashPasswd: encryptedDataWords.hash.toString(),
+      hashPasswd: encryptedDataWords.hash.key.toString(),
+      saltPasswd: encryptedDataWords.hash.salt,
```

Comment:
I think we should include both the hash algorithm and the number of iterations as well. The number of iterations may be a security parameter in the future, or we may need to change it later.

Reply:
Done